compaction/task_manager_module.hh

```diff
@@ -623,7 +623,9 @@ public:
         , _creator(std::move(creator))
         , _filter(std::move(filter))
         , _total_shard_size(total_shard_size)
-    {}
+    {
+        _total_shard_size = 0;
+    }
```

Comment:
we should initialize it in table_reshaping_compaction_task_impl::run() instead, and have shard_reshaping_compaction_task_impl::run() only add to it

Reply:
Done.